### ISSUE TITLE
not building in builds

### ISSUE THREAD

**Issue by variananora** [LinoxGH/MoreTools#23]
seems like builds page is using `build` branch to check stuff and the only branch that are active is only main.
https://thebusybiscuit.github.io/builds/LinoxGH/MoreTools/build/#10

**Comment by LinoxGH** [comment]
@variananora that's not an issue with MoreTools branches, the branch builds page looking for should be changed to `main`

**Comment by variananora** [comment]
yeah i posted this before i saw this https://github.com/TheBusyBiscuit/builds/pull/123
just need that to be approved
